specification/healthbot/resource-manager/Microsoft.HealthBot/stable/2020-10-20/healthbot.json

```diff
@@ -0,0 +1,630 @@
+        },
+        "type": {
+          "description": "the type of the bot for which availability needs to be checked",
+          "type": "string"
+        }
+      }
+    },
+    "CheckNameAvailabilityResponseBody": {
+      "description": "The response body returned for a request to check availability of a bot name.",
+      "type": "object",
+      "properties": {
+        "nameAvailable": {
+          "description": "indicates if the bot name is valid and available.",
+          "type": "boolean"
+        },
+        "reason": {
+          "description": "Reason for the result",
+          "type": "string"
+        },
+        "message": {
+          "description": "additional message from server",
+          "type": "string"
+        }
+      }
+    },
+    "Resource": {
+      "description": "Azure resource",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "description": "Specifies the resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "Specifies the name of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "location": {
+          "description": "Specifies the location of the resource.",
+          "type": "string"
+        },
+        "type": {
+          "description": "Specifies the type of the resource.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "Bot": {
+      "type": "object",
+      "description": "Bot resource definition",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/BotProperties",
+          "description": "The set of properties specific to healthcare bot resource."
+        }
+      }
+    },
+    "BotProperties": {
+      "title": "BotProperties",
+      "description": "The properties provided for bot creation",
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "description": "The display name of the bot."
+        },
+        "planId": {
+          "type": "string",
+          "description": "The plan ID of the bot."
+        },
+        "offerId": {
+          "type": "string",
+          "description": "The offer ID of the bot."
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "Gets the provisioning state of the Healthcare bot resource."
+        }
+      }
+    },
+    "ValidationResult": {
+      "title": "ValidationResult",
+      "description": "The response returned from validation process",
+      "properties": {
+        "status": {
+          "type": "string",
+          "description": "The status code of the response validation."
+        }
+      }
+    },
+    "Error": {
+      "title": "Error",
+      "description": "Bot error object.",
+      "properties": {
+        "status": {
```

Comment:
The error contract does not have a "status" property.  Please either refernce the common contract from specification\common-types\resource-management\v2\types.json or correct this to be identical.

Reply:
Fixed (referenced the common contract).